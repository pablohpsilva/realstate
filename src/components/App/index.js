import { version } from 'inferno'
import Logo from './logo'
import './App.css'

export default () => (
  <div className="App">
    <div className="App-header">
      <Logo width="80" height="80" />
      <h2>{`Welcome to Inferno ${version}`}</h2>
    </div>
    <p className="App-intro">
      To get started, edit <code>src/App.js</code> and save to reload.
    </p>
  </div>
)
