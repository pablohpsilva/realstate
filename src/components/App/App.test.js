import { render } from 'inferno';
import App from '@/App';

const createDiv = () => {
  return document.createElement('div');
}

expect.addSnapshotSerializer({
  test: (val) => val && val.children,
  print: (val) => `${val.children.toString()}`
})

it('renders without crashing', () => {
  const appRendered = render(<App />, createDiv());
  expect(appRendered).toMatchSnapshot()
});

it('renders a div with a className App-header', () => {
  const appRendered = render(<App />, createDiv());
  expect(appRendered.children[0].className).toBe('App-header')
  expect(appRendered).toMatchSnapshot()
})