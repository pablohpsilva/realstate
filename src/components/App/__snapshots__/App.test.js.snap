// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a div with a className App-header 1`] = `[object Object],[object Object]`;

exports[`renders without crashing 1`] = `[object Object],[object Object]`;
